Documentation/Contribution/Index.rst

```diff
@@ -0,0 +1,30 @@
+.. include:: /Includes.rst.txt
+
+============
+Contribution
+============
+
+Unlike most official TYPO3 documentation, this manual does not support the
+popular *Edit on GitHub* workflow for manual contributions, as these
+documentation files are automatically generated from the ViewHelper
+source files - and overwritten on each generation run.
+
+Therefore, contribute to this documentation by editing the appropriate source
```

Comment:
Actually we should link to the official TYPO3 Contribution Guide instead here. The GH->Gerrit workflow is only meant as a fallback not the main way to do things. Not sure if that fallback is/should be mentioned somewhere.